c2corg_ui/templates/xreport/view.html

```diff
@@ -0,0 +1,194 @@
+    ${show_badge(xreport, 'xreport')}
+  </h1>
+  <div class="outing-date" ng-cloak>${show_fulldate(xreport['date'])}</div>
+</header>
+
+% if xreport.get('areas'):
+  <article class="location-static text-center">
+    ${show_areas(xreport)}
+    % if xreport.get('elevation'):
+      <p class="text-center"><b>${xreport['elevation']} m</b></p>
+    % endif
+##     TODO SHOW COORDINATES ON THE YELLOW PAPER IN THE HEADER MAP
+##     % if geometry4326:
+##       <p>${round(geometry4326.x)} °E / ${round(geometry4326.y)} °N</p>
+##     % endif
+  </article>
+% endif
+
+<app-map class="view-details-map"></app-map>
+
+<section class="view-details-section images" app-view-details>
+  % if version:
+    ${show_archive_data('xreports', xreport, locale, version)}
+  % endif
+
+  ## IMAGES
+  % if not version:
+    ${get_image_gallery()}
+  % endif
+
+  <div class="view-details-informations col-xs-12 informations">
+    <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" data-toggle="collapse" data-target="#document-informations">
+      <span translate>Information</span><span class="glyphicon glyphicon-menu-down"></span>
+    </h3>
+    <section id="document-informations" class="collapse in">
+      ${get_xreport_general(xreport)}
+      % if not version:
+        ${get_xreport_participants(xreport)}
+        ${get_xreport_personal(xreport)}
+      % endif
+      ${get_licence(xreport)}
+    </section>
+  </div>
+
+  ${get_document_description(locale, 'xreport')}
+
+  <div class="description">
+    ${get_document_locale_text(locale, 'place')}
+    ${get_document_locale_text(locale, 'route_study')}
+    ${get_document_locale_text(locale, 'conditions')}
+    ${get_document_locale_text(locale, 'training')}
+    ${get_document_locale_text(locale, 'motivations')}
+    ${get_document_locale_text(locale, 'group_management')}
+    ${get_document_locale_text(locale, 'risk')}
+    ${get_document_locale_text(locale, 'time_management')}
+    ${get_document_locale_text(locale, 'safety')}
+    ${get_document_locale_text(locale, 'increase_impact')}
+    ${get_document_locale_text(locale, 'reduce_impact')}
+    ${get_document_locale_text(locale, 'modifications')}
+    ${get_document_locale_text(locale, 'other_comments')}
+  </div>
+
+  ## ASSOCIATIONS
+  % if not version:
+    <div class="view-details-associations col-xs-12 associations">
+      <span class="lead">
+##         <div ng-show="userCtrl.auth.isAuthenticated()" class="add-association">
+##           <label translate>Add association</label>
+##           <app-add-association parent-doctype="xreports" parent-id="${xreport_id}" dataset="wrc"></app-add-association>
+##         </div>
+        <section>
+          ${show_associated_waypoints(xreport)}
+          ${show_associated_routes(xreport)}
+          ${show_associated_articles(xreport)}
+          ${show_associated_outings(xreport)}
+        </section>
+      </span>
+    </div>
+  % endif
+
+  ${get_comments()}
```

Comment:
No comment in xreports. The "comments" float button should then be removed as well.